lib/associations/has-many.js

```diff
@@ -278,9 +278,9 @@ HasMany.prototype.injectSetter = function(obj) {
 
   obj[this.accessors.remove] = function(oldAssociatedObject, options) {
     var instance = this;
-    return instance[association.accessors.get]({
+    return instance[association.accessors.get](_.extend({
```

Comment:
Hi @mickhansen. Yes it is, but `_.extend()` is still supported as an alias. `_.extend()` is used all over the place in the Sequelize code base.

But do you want me to change it?